apps/grgsm_livemon.py

```diff
@@ -0,0 +1,391 @@
+#!/usr/bin/env python3
```

Comment:
Remove `grgsm_livemon.py` from the repository as it is generated/compiled from `grgsm_livemon.grc` during `build`

Reply:
Solved.